test/selenium/README.md

```diff
@@ -22,6 +22,62 @@ Run the [run-tests script](https://github.com/cockpit-project/cockpit/blob/maste
 
 Although this is the default way to run selenium tests the run-tests script is configurable and can be changed to run tests against different machines. This can be useful for developing or debugging tests. Check bellow the HACKING section for more details.
 
+### Debugging tests:
+Selenium testsuites via ``run-tests`` allows possibility how to debug tests.
```

Comment:
I don't really get what you mean with this sentence:
Of course you refer to `./test/selenium/run-tests` but not sure what about it.
If you want to say that when you run tests with './test/selenium/run-tests' you can debug these, you can write something like:
When running selenium tests with ``run-tests`` you can debug them with the following ways.